code/modules/reagents/reagent_containers/glass/bottle.dm

```diff
@@ -519,6 +519,16 @@ var/datum/disease2/disease/wizarditis = null
 	..()
 	reagents.add_reagent(ADMINORDRAZINE, 30)
 
+/obj/item/weapon/reagent_containers/glass/bottle/procizine
+	name = "Procizine Bottle"
```

Comment:
```suggestion
	name = "\improper Procizine bottle"
```

Reply:
Inconsistency with other bottles in file

Reply:
more like "old bad code does not excuse bad new code"

Reply:
Fixed